op-wheel/commands.go

```diff
@@ -92,6 +114,12 @@ var (
 	}
 )
 
+func withEngineFlags(flags ...cli.Flag) []cli.Flag {
+	return append(append(flags,
+		EngineEndpoint, EngineJWTPath, EngineOpenEndpoint, EngineVersion),
+		oplog.CLIFlags(envVarPrefix)...)
```

Comment:
why are the logger flags part of the engine?

Reply:
Because the logger is now always initialized for all `EngineAction`s. This is needed in turn because the `EngineAPIClient` and rpc client ctors needs a logger.